Packs/AWS-SecurityHub/Integrations/AWSSecurityHubEventCollector/AWSSecurityHubEventCollector.py

```diff
@@ -0,0 +1,205 @@
+import datetime
+
+import demistomock as demisto
+from AWSApiModule import *
+
+import urllib3
+import boto3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+VENDOR = 'AWS'
+PRODUCT = 'Security Hub'
+
+
+def get_events(client: boto3.client, start_time: datetime | None = None,
+               end_time: datetime | None = None, limit: int = 0) -> list[dict]:
+    """
+    Fetch events from AWS Security Hub.
+
+    Args:
+        client (boto3.client): Boto3 client to use.
+        start_time (datetime | None, optional): Start time to fetch events from. Required if end_time is set.
+        end_time (datetime | None, optional): Time to fetch events until. Defaults to current time.
+        limit (int): Maximum number of events to fetch. Can only be values between 0-100. Defaults to 0 (no limit).
+
+    Returns:
+        tuple[list, CommandResults]: A tuple containing the events and the CommandResults object.
+
+    """
+    kwargs = {}
+    _filters = {}
```

Comment:
why did you call it `_filters` and not `filters`?

Reply:
Probably started with `filter_` as `filter` is a reserved name, and then changed it to plural it and forgot to remove it.